packages/components/src/decision/Decision.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import Types from 'prop-types';
+import requiredIf from 'react-required-if';
+
+import SizeSwapper from '../sizeswapper';
+import NavigationOption from '../navigationOption';
+import Tile from '../tile';
+
+const DecisionType = { NAVIGATION: 'NAVIGATION' };
+const DecisionPresentantion = {
+  LIST: 'LIST',
+  LIST_BLOCK: 'LIST_BLOCK',
+  LIST_BLOCK_SMALL: 'LIST_BLOCK_SMALL',
+};
+
+const Decision = ({ options, presentation, type }) => {
+  if (type === DecisionType.NAVIGATION) {
+    const { LIST_BLOCK, LIST_BLOCK_SMALL } = DecisionPresentantion;
+    if (presentation === LIST_BLOCK || LIST_BLOCK_SMALL) {
```

Comment:
This won't work, I think you mean

```suggestion
    if (presentation === LIST_BLOCK || presentation === LIST_BLOCK_SMALL) {
```